src/client/editor/Editor.js

```diff
@@ -1419,11 +1350,17 @@ export default class Editor {
         object.remove(component._object);
       }
 
+      const animations = scene.animations;
+
+      if (animations.length > 0) {
+        setStaticMode(scene, StaticModes.Dynamic);
+      } else {
+        setStaticMode(scene, StaticModes.Static);
+      }
```

Comment:
does this mean that if there are any animations in the scene, we skip the combination for the whole scene?

Reply:
The whole loaded glTF scene, yes. I'm not sure how to make things safe with how ThreeJS manages animation paths.

Reply:
so if I have two models in my spoke scene, one has no animations, and the other does, the one without animations will still be combined? i guess that's OK -- seems like a tough tradeoff though.

Reply:
I'm confused, is this different from what you had in mind? Maybe you are confusing glTF scene and Spoke scene here? I'm not able to safely combine the ThreeJS object3Ds that are loaded from a glTF model with animations. It would require determining which object3Ds are included in the animation and should be left as dynamic.